x/ibc/light-clients/07-tendermint/types/client_state.go

```diff
@@ -201,23 +200,22 @@ func (cs ClientState) VerifyClientState(
 		return err
 	}
 
-	return merkleProof.VerifyMembership(cs.ProofSpecs, provingRoot, path, bz)
+	return merkleProof.VerifyMembership(cs.ProofSpecs, provingConsensusState.GetRoot(), path, bz)
 }
 
 // VerifyClientConsensusState verifies a proof of the consensus state of the
 // Tendermint client stored on the target machine.
 func (cs ClientState) VerifyClientConsensusState(
 	store sdk.KVStore,
 	cdc codec.BinaryMarshaler,
-	provingRoot exported.Root,
 	height exported.Height,
 	counterpartyClientIdentifier string,
 	consensusHeight exported.Height,
 	prefix exported.Prefix,
 	proof []byte,
 	consensusState exported.ConsensusState,
 ) error {
-	merkleProof, _, err := produceVerificationArgs(store, cdc, cs, height, prefix, proof)
+	merkleProof, provingConsensusState, err := produceVerificationArgs(store, cdc, cs, height, prefix, proof)
```

Comment:
nit: why don't we just return the `root` instead of full consensus state?

Another nit: can we change all `Verify...` functions to use variable `provingConsensusState` instead of `consensusState` for consistency?